src/frontend/src/pages/HomePage/components/Checklist.tsx

```diff
@@ -1,14 +1,17 @@
 import React, { useState } from 'react';
-import { Checklist as ChecklistType } from 'shared';
-import { Typography, Grid, Box, IconButton, useTheme } from '@mui/material';
+import { Checklist as ChecklistType, isAdmin } from 'shared';
+import { Typography, Grid, Box, IconButton, useTheme, Button } from '@mui/material';
 import { KeyboardArrowRight, KeyboardArrowDown } from '@mui/icons-material';
 import Task from './Task';
+import AddIcon from '@mui/icons-material/Add';
+import { useCurrentUser } from '../../../hooks/users.hooks';
 
 const Checklist: React.FC<{ parentChecklists: ChecklistType[]; checklistName?: string }> = ({
   parentChecklists,
   checklistName
 }) => {
   const theme = useTheme();
+  const currentUser = useCurrentUser();
```

Comment:
instead of checking the user is an admin like this, it would be better to pass in a prop from the AdminToolsOnboardingConfig file itself. Ther you should pass in the prop like `isAdmin={true}` and then in the checklist comopnent take in the prop called isAdmin and then use it from there